### DISCUSSION TITLE
added Italian contest 40/80

### DISCUSSION THREAD

**Discussion by ik3ocd** [mbridak/not1mm#125] [Ideas]
good morning, I'm Flavio ik3ocd, I would like to make a request if possible, to add to the contest list , contest italiano ari 40/80, thanks for your work contest linux program finally comfortable graphics.

**Comment by mbridak** [comment]
@ik3ocd Done.

**Comment by ik3ocd** [comment]
hi michael, good news and great work as always thanks for having inserted
contest ita 40/80 ari, I think it is the first insertion ever in a program
for Linux and this does you credit, if you like I will spread among my
acquaintances of this insertion of yours. Bye and best regards 73.

Il Ven 11 Apr 2025, 03:16 Michael Bridak ***@***.***> ha
scritto:
